crates/rpc/rpc/src/eth/api/traits/blocking_task.rs

```diff
@@ -0,0 +1,15 @@
+//! Spawns a blocking task. Should be used for long-lived database reads.
```

Comment:
where is this doc coming from?

long lived db reads should be avoided. 
this is not the motivation for this trait, this is for any tasks that are blocking.

please remove long-lived from the doc and add smth like `any blocking IO`

Reply:
in the context of rpc, long-lived reads from db are the most common case of blocking io, no?